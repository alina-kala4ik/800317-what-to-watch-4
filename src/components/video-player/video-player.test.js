import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

it(`render VideoPlayer`, () => {
  const tree = renderer
    .create(<VideoPlayer
      videoPreview={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
      src={`img/fantastic-beasts-the-crimes-of-grindelwald.jpg`}
      isPlaying={false}
    />, {
      createNodeMock: ()=>{
        return {};
      }
    })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
