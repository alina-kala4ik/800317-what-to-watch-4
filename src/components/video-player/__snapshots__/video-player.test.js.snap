// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render VideoPlayer 1`] = `
<video
  autoPlay={false}
  height="175"
  loop={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  preload="none"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width="280"
/>
`;
